# PoseNames.msg
string[] names  # Array of pose names
